### PR TITLE
Implement remained natives for Linux (closes #12)

### LINKED ISSUE #12: Support Linux
- [x] getInstallations 
- [x] findWindow
- [x] open
- [x] close
- [ ] screenshot
- [ ] resize
- [ ] getWindowSize
- [ ] generateThumbnail

### PR DESCRIPTION
**Pull request by AndreyBelym**
\cc @AlexanderMoskovkin @georgiy-abbasov

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "testcafe-browser-tools",
-  "version": "1.0.0",
+  "version": "1.1.0",
```
Comment (AlexanderMoskovkin) [inline package.json]: There are no any changes in public API, so it's not a minor change. 
More than that I suppose we are not going to release a new version with only this enhancement. Maybe it's better to implement some more features for linux and then increase the version?

**Comment by AndreyBelym** [comment]
FPR

**Comment by AlexanderMoskovkin** [comment]
lgtm

**Comment by AndreyBelym** [comment]
FPR, all natives are done, tested in sandbox with Ubuntu 14.04 LTS 32bit and Ubuntu 16.04 64bit inside VirtualBox.
\cc @AlexanderMoskovkin @georgiy-abbasov

**Comment by inikulin** [comment]
@AndreyBelym I believe it's worth checking with other popular repos

**Pull request merged**